### PR TITLE
Update Comparison API version.

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-comparison/GroupDocs.Comparison-for-.NET-MVC/pull/30.

Related PR: https://github.com/groupdocs-total/GroupDocs.Total-Angular/pull/173.

### CODE REVIEW

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451796&bid=17874270)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 14
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451797&bid=17874271)

File: src/Products/Comparison/Service/ComparisonServiceImpl.cs [hunk 1]
```diff
@@ -110,250 +97,175 @@ public bool CheckFiles(CompareRequest files)
...
-                throw new Exception("Something went wrong. We've got null result.");
+                comparer.Compare(outputStream, compareOptions);
             }
-            return compareResult;
+
+            return comparer;
         }
 
-        private CompareResultResponse GetCompareResultResponse(ICompareResult compareResult, ChangeInfo[] changes, string ext)
+        private CompareResultResponse GetCompareResultResponse(ChangeInfo[] changes, string resultGuid)
```
Comment (bobkovalex) [inline src/Products/Comparison/Service/ComparisonServiceImpl.cs:257]: Codacy Issue found: [Make 'GetCompareResultResponse' a static method.](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451797)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451796&bid=17874270)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 14
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451797&bid=17874271)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 14
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451797&bid=17874271)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451796&bid=17874270)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 14
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451797&bid=17874271)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 10
           

Complexity increasing per file
==============================
- src/Products/Comparison/Service/ComparisonServiceImpl.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-WebForms/pullRequest?prid=5451796&bid=17874270)

**Review by Pavel-Teplitsky — APPROVED** [review]

**Pull request merged**